testsuite/integration-arquillian/test-apps/js-console/example-realm.json

```diff
@@ -68,7 +68,8 @@
             "webOrigins": [
                 "http://localhost:8280",
                 "https://localhost:8643"
-            ]
+            ],
```

Comment:
I would have sworn I needed this for some reason. But maybe I mixed it with something else. Thanks for notice.